### ISSUE TITLE
calculate relative paths

### ISSUE THREAD

**Issue by FreakinaBox** [gwuhaolin/web-webpack-plugin#4]
current code seems to just copy and paste the output filename into the script tag. need some logic added to calculate relative path based on js location and HTML location.

```
{
    ....
    output: {
        filename: "./js/dist/[name]-[chunkhash].min.js"
    },

    plugins: [
        ....
        new WebPlugin({
            template: 'js/angular/pages/reports/index.html',
            filename: 'org/index.php',
            requires: ['common', 'reports'],
        })
    ]
}
```
plugin outputs
```
<script src="js/dist/common-9359363eb6cc11f14a53.min.js"></script>
<script src="js/dist/reports-c55beadeae38133a9cd5.min.js"></script>
```
should be
```
<script src="../js/dist/common-9359363eb6cc11f14a53.min.js"></script>
<script src="../js/dist/reports-c55beadeae38133a9cd5.min.js"></script>
```

**Comment by gwuhaolin** [comment]
can you provide you project or output dir struct?
I'am not sure what you say.

**Comment by FreakinaBox** [comment]
the resulting output is
```
/org/index.php
/js/dist/common-9359363eb6cc11f14a53.min.js
/js/dist/reports-c55beadeae38133a9cd5.min.js
```

**Comment by gwuhaolin** [comment]
`filename: 'org/index.php'` in `new WebPlugin()` lead to this error output.
If you change it to 'index.php'` output will be fine.

**Comment by gwuhaolin** [comment]
`filename: 'org/index.php'` means output html file's name not path. you can't use filename like `filename: 'org/index.php'`. If you change it to `index.php` output will be fine.

I can't support this scenes because calculate relative paths will bring a new issue for set `publicPath: '//js.cdn'` like [this demo](https://github.com/gwuhaolin/web-webpack-plugin/tree/master/demo/public-path).

**Comment by FreakinaBox** [comment]
Thank you for making that change. I am testing now.
